guides/node-operators/run-a-super-world-computer-node.md

```diff
@@ -2,89 +2,121 @@
     ```
 
 ---
 
 ## Devnet
 
-
 ### Steps
 
-1. Prepare [rollup.json](https://github.com/ethstorage/pm/blob/main/L2/assets/devnet_rollup.json) and [genesis.json](https://github.com/ethstorage/pm/blob/main/L2/assets/devnet_genesis.json).
+1. Follow the following steps to build a node. The steps are basically the same as in [Optimism's documentation](https://docs.optimism.io/builders/node-operators/tutorials/node-from-source), the only difference is that here we use the `devnet` branch of both [our optimism fork](https://github.com/ethstorage/optimism/tree/devnet) and [our op-geth fork](https://github.com/ethstorage/op-geth/tree/devnet) instead.
 
-2. Follow the steps [here](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup) to build `op-node` and `op-geth`, the only difference is: use the `devnet` branch of both [optimism](https://github.com/ethstorage/optimism/tree/devnet) and [op-geth](https://github.com/ethstorage/op-geth/tree/devnet) instead.
+- 1.1 Build `op-geth`
+    ```bash
+    git clone -b devnet https://github.com/ethstorage/op-geth.git
+    cd op-geth && make geth
+    ```
+- 1.2 Build `op-node`
+    ```bash
+    cd ..
+    git clone -b devnet https://github.com/ethstorage/optimism.git
+    cd optimism && make op-node
+    ```
 
-3. Setup `op-node` and `op-geth` following the steps below:
+2. Setup `op-geth`:
 
     ```bash
         # assume optimism and op-geth repo are located at ./optimism and ./op-geth
 
-        cd op-geth
+        cd ../op-geth
+        # prepare devnet_genesis.json
+        curl -LO https://raw.githubusercontent.com/ethstorage/pm/main/L2/assets/devnet_genesis.json
 
-        build/bin/geth init --datadir=datadir genesis.json
+        build/bin/geth init --datadir=datadir devnet_genesis.json
 
         openssl rand -hex 32 > jwt.txt
 
         # We don't specify `--rollup.sequencerhttp` since it's for testing blob archiver only.
         # The rpc port is the default one: 8545.
         ./build/bin/geth   --datadir ./datadir   --http   --http.corsdomain="*"   --http.vhosts="*"   --http.addr=0.0.0.0   --http.api=web3,debug,eth,txpool,net,engine   --ws   --ws.addr=0.0.0.0   --ws.port=8546   --ws.origins="*"   --ws.api=debug,eth,txpool,net,engine   --syncmode=full   --gcmode=archive   --nodiscover   --maxpeers=0   --networkid=42069   --authrpc.vhosts="*"   --authrpc.addr=0.0.0.0   --authrpc.port=8551   --authrpc.jwtsecret=./jwt.txt   --rollup.disabletxpoolgossip=true --enablel2blob
+    ```
+ 
+ 3. Setup `op-node`:
+    ```bash
+        # copy jwt.txt from the op-geth directory above to optimism/op-node
+        cp jwt.txt ../optimism/op-node 
```

Comment:
This is assuming `cwd` is `./op-geth`, I feel it's better to use the same assumption above: "assume optimism and op-geth repo are located at ./optimism and ./op-geth".